ecoli/processes/antibiotics/permeability.py

```diff
@@ -7,6 +7,7 @@
 from vivarium.plots.simulation_output import plot_variables
 from vivarium.processes.timeline import TimelineProcess
 
+# TODO: Recompute average surface area with corrected formula 
```

Comment:
I've made a couple comments to this effect elsewhere, but I realized that the surface area we've been using was probably calculated on a very old commit (perhaps even before we realized we were using $3\pi r^2$ instead of $4 \pi r^2$).